src/t_stream.c

```diff
@@ -2122,7 +2141,7 @@ void xpendingCommand(client *c) {
             raxStop(&ri);
         }
     }
-    /* XPENDING <key> <group> <start> <stop> <count> [<consumer>] variant. */
+    /* XPENDING <key> <group> [IDLE <idle>] <start> <stop> <count> [<consumer>] variant. */
```

Comment:
the syntax here is different than the one at the top (no nesting), better fix it, or just make the comment more generic. i.e. something in the lines of `<start>,<stop>,<count> provided, return actual pending entries (not just info)`